main.js

```diff
@@ -113,18 +113,22 @@ function savePalette() {
 }
 
 function isPaletteUnique(palettesList, singlePalette) {
-    for (i = 0; i < palettesList.length; i++) {
-        var matches = true;
-        for (j = 0; j < palettesList[i].length; j++) {
-            if (palettesList[i][j] !== singlePalette[j]) {
-                matches = false;
-                break;
-            }
-        }
-        if (matches) {
+    for (var i = 0; i < palettesList.length; i++) {
+        console.log('isPaletteUnique', i);
+        if (areArraysEquivalent(palettesList[i], singlePalette)) {
```

Comment:
<img width="21" alt="image" src="https://user-images.githubusercontent.com/123283073/232258686-2a98f005-b384-4c7b-a36a-02be60a7c2e7.png"> I did not know we could invoke a function inside a conditional to get a true or false. Amazing!!!

Reply:
me neither woo!